src/config/delegation.cpp

```
@@ -0,0 +1,102 @@
+    } else {
+      target.host = host_port;
+    }
+  } else {
+    // Just model name — use current host
+    target.model = value;
+  }
+  return target;
+}
+
+/// Load delegation config from file. Simple YAML subset: "  key: value"
+static std::unordered_map<std::string, DelegationTarget> load_config() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)